nginx/etc/nginx/conf.d/api.conf

```diff
@@ -22,6 +24,16 @@ server {
         proxy_pass http://api-server-upstream;
     }
 
+    location ~ ^\/api\/exports\/.*\/files\/.* {
```

Comment:
It looks like a file name could be part of this request URI. If so, I think that we want to make this match with `~*` vs. `~` so that it catches things in a case-insensitive way (e.g., `.TIF` and `.tif`).